travel-event-host-app/src/components/header/Header.tsx

```diff
@@ -8,15 +8,16 @@ import { useState } from 'react';
 import AuthDialog from '../auth-dialog/AuthDialog';
 import HeaderBarAvatar from '../avatar/header-bar-avatar/HeaderBarAvatar';
 import styles from './styles.module.css';
+import { signOut } from 'next-auth/react';
+import { useOnboardingContext } from '@/lib/context';
+
 export default function Header() {
+  const { session, status } = useOnboardingContext();
```

Comment:
What does useOnboardingContext mean?

Reply:
is the hook to get variables from the context, I explain it on the discord message